### ISSUE TITLE
Cannot resolve symbol Builder

### ISSUE THREAD

**Issue by crysan** [TheFinestArtist/FinestWebView-Android#3]
Hi!

I use in the fragment:
new FinestWebViewActivity.Builder(getActivity()).show(author.getUrl_base());

error:
Cannot resolve symbol Builder

**Comment by TheFinestArtist** [comment]
Hello, @crysan you have to use FinestWebView.Builder() instead FinestWebViewActivity.Builder()

**Comment by TheFinestArtist** [comment]
Updated README 71c251c53fba6f8edb351e60f0d472bc727a674d

**Comment by crysan** [comment]
And!
How to apply:
overridePendingTransition(R.anim.modal_activity_open_enter, R.anim.modal_activity_open_exit);

cannot resolve method 'overridePendingTransition'

**Comment by TheFinestArtist** [comment]
@crysan Thanks for your suggestion, I was actually made this method `overridePendingTransition` or 'setTransition` with all 4 animation. But I found I need current activity to call overridePendingTransition method. 

So there is pros & cons.
If you add `overridePendingTransition` new method, I have to get current activity not context.
If I don't then developer has to call `overridePendingTransition` every time.

Which one do you this is better?

**Comment by TheFinestArtist** [comment]
@crysan Which way do you prefer?

1 Current Version

``` java
new FinestWebView(Context).setCloseAnimations(int anim, int anim).show();
overridePendingTransition(int anim, int anim);
```

2 Your Suggestion (Get activity first time)

``` java
new FinestWebView(Activity).setCustomAnimations(int anim, int anim, int anim, int anim).show();
```

3 Your Suggestion (Get context first time and add activity for animation call)

``` java
new FinestWebView(Context).setCustomAnimations(Activity activity, int anim, int anim, int anim, int anim).show();
```

**Comment by crysan** [comment]
Option 2 is better.

However, I was not talking about it.
When you add a line:
overridePendingTransition (R.anim.modal_activity_open_enter, R.anim.modal_activity_open_exit);
system says:
can not resolve method 'overridePendingTransition'

Those.  there is no such method.

**Comment by TheFinestArtist** [comment]
@crysan You have to use this `overridePendingTransition` method in your activity.

**Comment by TheFinestArtist** [comment]
@crysan http://developer.android.com/reference/android/app/Activity.html#overridePendingTransition(int, int)

**Comment by crysan** [comment]
Stop.Stop.Stop.
http://firepic.org/images/2015-12/02/vui5k2v1kv8z.png

**Comment by TheFinestArtist** [comment]
@crysan Change your method to get Activity instead of Context. And call overridePendingTransition from your activity object.

``` java
public void openLinkBrowser(String url, int type, Activity activity) {
  // ...
  new FinestWebView.Builder(activity)
    .show(...);
  activity.overridePendingTransition(R.anim.xxx, R.anim.xxx);
  // ...
}
```

And you don't have to call `overridePendingTransition` if you don't want any animation!

I really should change this API. I think `overridePendingTransition` is confusing.
`new FinestWebView(Activity).setCustomAnimations(int anim, int anim, int anim, int anim).show();`

**Comment by crysan** [comment]
Good! Ty!

**Comment by TheFinestArtist** [comment]
@crysan Welcome :)

**Comment by crysan** [comment]
And waiting version
compile 'com.thefinestartist:finestwebview:1.0.1'

**Comment by TheFinestArtist** [comment]
Yeah, it will be 1.0.1

I will let you know if I'm done. I'm still choosing which way looks better.

**Comment by crysan** [comment]
Ok

**Comment by TheFinestArtist** [comment]
@crysan Released new version [1.0.2](https://github.com/TheFinestArtist/FinestWebView-Android/releases/tag/1.0.2)
